firefox-ios/Client/Frontend/Library/Bookmarks/Edit Folder/EditFolderViewModel.swift

```diff
@@ -92,17 +96,23 @@ class EditFolderViewModel {
         }
     }
 
-    func save() {
-        guard let folder else { return }
+    @discardableResult
+    func save() -> Task<Void, Never>? {
```

Comment:
Same, this returns a `Task` so we can test this async code without completion handlers